components/asana/sources/task-updated-in-project/task-updated-in-project.mjs

```diff
@@ -64,16 +76,18 @@ export default {
 
       const responses = await Promise.all(promises);
 
-      responses.forEach(({
-        event, task,
-      }) => {
-        const ts = Date.parse(event.created_at);
-        this.$emit(task, {
-          id: `${task.gid}-${ts}`,
-          summary: task.name,
-          ts,
+      responses
+        .filter(({ task }) => user.length === 0 || task.assignee && task.assignee.gid === user)
```

Comment:
I would do a _falsy_ check for `user` instead of checking its length - as an optional prop that is not set could very well be `undefined`
```suggestion
        .filter(({ task }) => !user || task.assignee && task.assignee.gid === user)
```